src/components/Assets/UploadAssetsComponent.jsx

```diff
@@ -0,0 +1,111 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
+/* eslint-disable react/prop-types */
+/* eslint-disable react/no-unused-prop-types */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Progress, Icon } from 'semantic-ui-react';
+import Dropzone from 'react-dropzone';
+import '../../_css/UploadAssets.css';
+
+const StatusMessage = props => <div className={`${props.className} `}>{props.message}</div>;
+
+const uploadStatus = (success, error) => {
+  if (success.hasOwnProperty('fail') || error) {
+    const errorIcon = (
+      <div>
+        <Icon name="x icon" size="big" />
+        {success.fail || error}
+      </div>
+    );
+    return errorIcon;
+  }
+  const successIcon = (
+    <div>
+      <Icon name="check" size="big" />
+      {success.success}
+    </div>
+  );
+  return successIcon;
+};
+
+const errorMessageHelper = (error, success, handleFileDownload) => {
+  if (success.hasOwnProperty('fail')) {
+    return (
+      <span className="error-guide">
+        Please download
+        <a href="#" onClick={() => handleFileDownload(success.file)}>
+          this file
+        </a>{' '}
+        , fix errors and upload again.
+      </span>
+    );
+  }
+
+  if (error) {
+    return (
+      <span className="error-guide">
+        Please confirm the file is well formatted and try uploading again.
+      </span>
+    );
+  }
+
+  return null;
```

Comment:
Return early. That is:
```jsx
if (!success.hasOwnProperty('fail') || !error) {
  return null;
}

if (success.hasOwnProperty('fail')) {
  return (...);
}

return (
  <span className="error-guide">
    Please confirm the file is well formatted and try uploading again.
  </span>
);

Reply:
This [comment](https://github.com/AndelaOSP/art-dashboard/pull/177/files#r239454380) has not been addressed